yarn.lock

```diff
@@ -953,6 +975,11 @@ ansi-styles@^5.0.0:
   resolved "https://registry.yarnpkg.com/ansi-styles/-/ansi-styles-5.2.0.tgz#07449690ad45777d1924ac2abb2fc8895dba836b"
   integrity sha512-Cxwpt2SfTzTtXcfOlzGEee8O+c+MmUgGrNiBcXnuWxuFJHe6a5Hz7qwhwe5OgaSYI0IJvkLqWX1ASG+cJOkEiA==
 
+ansi-styles@^6.1.0:
```

Comment:
=> Found "wrap-ansi#ansi-styles@6.2.1"
info This module exists because "tsup#sucrase#glob#jackspeak#@isaacs#cliui#wrap-ansi" depends on it.